fabric/bin/peer.sh

```diff
@@ -1,4 +1,4 @@
-#!/bin/bash
+#!/bin/bash -vx
```

Comment:
Do you really wanna keep this `-vx`? It spams my terminal.

Reply:
Can here also please add a short title to the commit message. Thank you

Reply:
removed -vx (leftover from debugging) and added short title to commit list

Reply:
thanks